color_refiment_helper.py

```diff
@@ -8,51 +8,66 @@
 
 
 def debug(*args):
+    """
+    Prints debug statements when DEBUG is set to `True`
+
+    :param args: argument to be printed
+    :return:
+    """
     if DEBUG:
         print(*args)
 
 
-def compare(s: "List", t: "List"):
+def compare(s: "List", t: "List") -> bool:
+    """
+    Compares if two lists are equal
+
+    :param s: List to compare
+    :param t: List to compare
+    :return: Returns `True` if list have the same length and contain the same elements. Elements do not need to have the
+    same order. Returns `False` otherwise.
+    """
+    # TODO: change internal comparison since `Counter` may not be used
     return Counter(s) == Counter(t)
 
 
-def create_partition(old_coloring: "Coloring", vertex1: "Vertex", vertex2: "Vertex") -> "Coloring":
+def create_partition(coloring: Coloring, vertex1: Vertex, vertex2: Vertex) -> Coloring:
     """
-    Returns a new coloring where both vertices are in a new partition and removed from the one they belonged to.
+    Returns a new coloring where both vertices have the same new color and are removed from the one they belonged to
 
```

Comment:
Hier zeg je have the same color terwijl je hier onder zegt separate color. Dus ik vind dit nog niet helemaal duidelijk

Reply:
Wat ie doet is de gegeven vertices aan een nieuwe color class toevoegen. Hiermee hebben ze dezelfde kleur, maar seperate van de rest van de vertices.

Reply:
Dat ga ik dus nog maar even verduidelijken;)